tests/console/python3_beautifulsoup4.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: python3-beautifulsoup tests
+# - install python-beautifulsoup package
+# - use library to parse sample htmlfile
+# - Compare the result vs expected
+#
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'zypper_call';
+use python_version_utils;
```

Comment:
I think `python_version_utils`, needs to be put into the other modules that use similar handling

Reply:
Yes, my idea is to do that as a two step process. First merge this PR and introduce the common library, without need to re-test existing code. 

As second step, since there is already a [proposal]( https://progress.opensuse.org/issues/138401?issue_count=23&issue_position=1&next_issue_id=138593#note-2) for refactoring all the other python-related test modules, we can take the occasion to group duplicate functionality in the same library because we will nevertheless need to do VR on the refactored tests.